### DISCUSSION TITLE
environment variables

### DISCUSSION THREAD

**Discussion by karlglazebrook** [iraf-community/iraf#163] [Q&A] [answered]
Got this working on an M1 Mac using community iraf. The python was from anaconda and was intel! (Even though the iraf was a native build)

I found for PyRAF to work after build/install I had to do 
```
export iraf=/Users/karl/GIT/iraf
export IRAFARCH=macos64

```
However this prevents 'ecl' from working! Not sure what the answer is here. ecl gives the error:
```
/Users/karl/GIT/irafbin.macos64/ecl.e not found
/Users/karl/.iraf/bin/ecl: line 156: /Users/karl/GIT/irafbin.macos64/ecl.e: No such file or directory
/Users/karl/.iraf/bin/ecl: line 156: exec: /Users/karl/GIT/irafbin.macos64/ecl.e: cannot execute: No such file or directory

```

I also note it gave a warning about aqutil not being available.

**Comment by karlglazebrook** [comment]
PS I was using python 3.8. I was surprised and thought pyraf might only work with python2?

**Comment by olebole** [comment, marked as answer]
You need a slash at the end of $iraf:
```
export iraf=/Users/karl/GIT/iraf/
```

**Comment by karlglazebrook** [comment]
Yep that does the trick!

Karl
